src/sidebar/app.js

```diff
@@ -37,6 +37,26 @@ if (settings.raven) {
   angular.module('ngRaven', []);
 }
 
+if(settings.googleAnalytics){
```

Comment:
I've tested this and verified that analytics reporting works in the context of the browser extension. Can we move this to a function in a separate module (eg. `analytics.js`) and just add a note that the minified block at the top is the minified GA tracking snippet.